script/generateDocs.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+## Build package
+mvn package -DskipTests=true
```

Comment:
Why not turn this into a mvn goal or something?

Reply:
Yea thought about that.  We could call out to this, or add a bunch of goals for each config type, dunno which is better

Reply:
I would do one goal in the root POM that takes a comma delimited list of classes or something like that.  That way it's injected at that base POM.

Reply:
Hooked this into the site goal.  IE `mvn clean site`

Didn't feel like its worth running on every compile